flake.nix

```diff
@@ -3,40 +3,62 @@
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
-    flake-utils.url = "github:numtide/flake-utils";
     ixx = {
       # match version with npm package
       url = "github:NuschtOS/ixx/v0.0.6";
       inputs = {
         nixpkgs.follows = "nixpkgs";
-        flake-utils.follows = "flake-utils";
```

Comment:
Since we are already deduping the input here I think the extra input is not that big of an issue. We are only using one utility library and as long as we are staying at one I think it is not such big of a problem.